data-raw/syn_data_creation.R

```diff
@@ -2,15 +2,20 @@
 #' @export
 #' @rdname syn_test_data
 syn_test_data <- function() {
-  sd <- scda::synthetic_cdisc_data("rcd_2022_06_27")
-  sd$adhy <- NULL
-  sd$adpp <- NULL
-  sd$adqs <- NULL
-  sd$adtr <- NULL
-  sd$adpc <- NULL
-  sd$adab <- NULL
-  # to avoid bug
-  attr(sd, "data_from") <- NULL
```

Comment:
This attribute was passed by `scda` and apparently it was causing some issues when we were still using the `dm` package. Usage of `dm` has been removed and this attribute is anyway not populated anymore by `random.cdisc.data` since we are creating our own list.